rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -100,8 +101,8 @@ def persist(self, model_dir):
         # type: (Text) -> Dict[Text, Any]
         file_name = self.name+".json"
         full_name = os.path.join(model_dir, file_name)
-        with io.open(full_name, 'w') as f:
-            f.write(json.dumps({"dimensions": self.dimensions}))
+            with io.open(full_name, 'w', encoding="utf-8") as f:
```

Comment:
Two things:
- there is a missplaced space infront of `with`
- I'd rather leave this code as it was and instead directly ensure that the `self.dimensions` array only contains uncode strings. Since that array is either created by passing it in directly (where we already make sure it is unicode) or by setting it with the `available_dimensions()` method. So what I suggest is, revert this part of the change and in the `available_dimensions` function wrap the `m[1]` with str as `str(m[1])` while importing `from builtins import str`